### PR TITLE
Code refactor - removed duplication

### PR DESCRIPTION
**Pull request by adityachandak287**
Removed code duplication in comprehend-service worker.

### CODE REVIEW

File: comprehend-service/worker.js [hunk 1]
```diff
@@ -87,29 +87,24 @@ if (process.env.NODE_ENV !== "test") {
     };
 
     //console.log(msg);
+
+    var params = {
+      LanguageCode:
+        TweetObj.lang in langComprehend
+          ? TweetObj.lang
+          : "en" /* required - other option is es */,
+    };
```
Comment (adityachandak287) [inline comprehend-service/worker.js:96]: `params` can be declared just once.

File: comprehend-service/worker.js [hunk 2]
```diff
@@ -87,29 +87,24 @@ if (process.env.NODE_ENV !== "test") {
...
-      var params = {
-        LanguageCode:
-          TweetObj.lang in langComprehend
-            ? TweetObj.lang
-            : "en" /* required - other option is es if not supported run for english*/,
-        Text: msg.full_text,
-      };
-      TweetObj.text = msg.full_text;
+      params.Text = msg.full_text;
     }
```
Comment (adityachandak287) [inline comprehend-service/worker.js:104]: `Text` property can be changed according to retweet status

Comment (adityachandak287) [inline comprehend-service/worker.js:104]: `Text` property can be changed according to retweet status

**Comment by YashMeh** [comment]
Why did not I get a notification for this o.O

**Pull request merged**